### PR TITLE
Rust ecosystem

### PR DESCRIPTION
**Pull request by lpil**
### Checklist

* [x] I have read the [Contributor Guide](../../CONTRIBUTING.md)
* [x] I have read and agree to the [Code of Conduct](../../CODE_OF_CONDUCT.md)
* [x] I have added a description of my changes and why I'd like them included in the section below

### Description of Changes

Ensure that Rust projects are listed in the Rust Ecosystem repo

### Related Issues

https://github.com/EmbarkStudios/opensource/issues/22

### CODE REVIEW

File: src/validate/context.rs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    pub rust_ecosystem_readme: String,
+}
+
+impl Context {
+    pub async fn get() -> eyre::Result<Self> {
+        let embark_github_organisation_members =
+            github::public_organisation_members("EmbarkStudios").await?;
+
+        let rust_ecosystem_readme =
+            github::download_repo_file("EmbarkStudios", "rust-ecosystem", "main", "README.md")
```
Comment (soniasingla) [inline src/validate/context.rs:21]: Don't we need to add the `owners` also here?

**Review by soniasingla — CHANGES_REQUESTED** [review]
Looks amazing, thanks @lpil 😻💪🏻

File: src/validate/context.rs (hunk 1 above)
Comment (lpil) [reply src/validate/context.rs:21]: The CODEOWNERS file? That's a different file per project so we don't download it ahead of time. This Context information is shared between all the projects.

**Review by soniasingla — APPROVED** [review]

**Comment by soniasingla** [comment]
over to @arirawr for final review 👋🏻

**Review by arirawr — APPROVED** [review]

**Comment by lpil** [comment]
Thanks gang

**Pull request merged**